types/openlayers/v3/index.d.ts

```diff
@@ -63,19 +63,13 @@ declare module ol {
       * Error object thrown when an assertion failed. This is an ECMA-262 Error,
       * extended with a `code` property.
       * @see {@link https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Error}
-      * @constructor
-      * @extends {Error}
-      * @implements {oli.AssertionError}
```

Comment:
this `implements` is missing from the type

Reply:
I ended up reverting this back to `master`. +1 to fixing it up, but I'm running low on time budget...